payu/experiment.py

```diff
@@ -794,6 +794,10 @@ def archive(self, force_prune_restarts=False):
                 payu=self.payu_path,
                 expt=self.counter
             )
+
+            if not self.metadata.enabled:
+                cmd += f' --metadata-off'
+
```

Comment:
Would it be possible, and therefore easier, to propagate this option by setting the `PAYU_METADATA_OFF` environment variable?

Reply:
Yes it is possible! I've added it in f89bff0. Only error I can see is when running commands like `payu-run` say that spins up a `collate`/`sync` job - would that be an issue?

Reply:
Ok resolving this as removed this option from `run`/`sync`/`collate` commands